app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,59 @@
+<ul class="nav d-flex justify-content-between bg-primary mb-0" style="width:100%">
+  <li class="nav-item">
+    <%= link_to(
+                  raw("<i class='fas fa-arrow-left'></i>"),
+                  groups_path,
+                  class: "nav-link text-light border-0"
+                )%>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link text-light" href="#">TRANSACTIONS</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link text-light" href="#"><i class="fas fa-search"></i></I></a>
+  </li>
+</ul>
```

Comment:
- [ ] Please consider doing the same with this view:

![image](https://github.com/Elhambasir/budget-app/assets/77069011/251a8ece-dc22-47b4-88fe-eceba97fbde4)

The button should be at the bottom! 🤓 🔧